node/store.go

```diff
@@ -539,6 +513,50 @@ func (s *Store) StoreBlobs(ctx context.Context, blobs []*core.BlobMessage, blobs
 	return &keys, nil
 }
 
+func (s *Store) StoreBatchBlobMapping(ctx context.Context, batchHeaderHash [32]byte, blobHeaderHashes [][32]byte) error {
+	start := time.Now()
+	// The key/value pairs that need to be written to the local database.
+	keys := make([][]byte, 0)
+	values := make([][]byte, 0)
+
+	expirationTime := s.expirationTime()
+	expirationKey := EncodeBatchExpirationKey(expirationTime)
```

Comment:
We allow multiple blobs to have the same expiration time, but not for batches. Is this correct? Esp. if we were to allow multiple dispersers?

Reply:
Yes, a blob have its [own expiration time](https://github.com/Layr-Labs/eigenda/blob/master/node/store.go#L424). This would just expire the batch <-> blob mapping.

Reply:
What if there happen to be two batches that get created with the same expiration time? It seems like we're worrying about that case for blobs but not for batch mappings.

Reply:
I see what you mean. That's a valid edge case. 
In fact, the existing `StoreChunks` endpoint would have this issue as well. 

I also see an issue with how we're adding expirations for blobs - there could be a concurrency issue that can void a write without a lock. 
I'll refactor the expiration key such that the expiration key is in format: 
- `_EXPIRATION_<EXPIRATION_TIME>_<BATCH_HEADER_HASH>` -> batch header hash
- `_BLOBEXPIRATION_<EXPIRATION_TIME>_<BLOB_HEADER_HASH>` -> blob header hash

This way we can still query expirations by expiration time, but each batch/blob gets its own record. Wdyt? 

cc @jianoaix